defi/amaralend.md

```diff
@@ -0,0 +1,90 @@
+---
+title: AmaraLend
+description: AmaraLend Beta is a top lending market providing lending service for all holders within Polkadot ecosystem.
+---
+
+# AmaraLend - An Multi-Chain Lending Platform on Moonbeam
+
+![Amaralend Banner](../images/amaralend/dapps-amaralend-banner.png)
+
+**Disclaimer:** Projects themselves entirely manage the content in this guide. Moonbeam is a permissionless network. Any project can deploy its contracts to Moonbeam.
+
+## Introduction
+
+AmaraLend Beta is a top lending market providing lending service for all holders within Polkadot ecosystem.
+
+AmaraLend is a multi-chain protocol planning to deploy on every top-notch parachain within Polkadot.
+
+Moonbeam is fully compatible with Ethereum EVM and Substrate, which is friendly to developers and the best start for them to enter the Polkadot ecosystem.The first Polkadot parachain AmaraLend deployed on is Moonbeam. Currently, AmaraLend Beta has been on Moonbase Alpha for testing and after which it will be officially deployed on the Moonriver network.
+
+**AmaraLend complete version will be featured:**
+
+- **Multi-chain Deployment：**AmaraLend plans to realize multi-chain deployment on Polkadot top-notch parachain.
+- **Graded Market**: Amara develops both the mainstream assets market and innovation market. The long-tail quality assets will be included in the innovation market to fully release the liquidity. The AmaraLend lending market embraces popular assets within Polkadot.
+- **NFT Incentive system：**Integrae NFT into community positive incentive system to reward active members.
+- **Cross-chain Bridge：**Amara has developed multi-signature aseet cross-chain protocol MaraLink to introduce popular public chains into Polkadot ecosystem. Please refer to our official website for a detailed introduction. We will also list MaraLink on Moonbeam docs for open use.
+
+You can read more about AmaraLend in the following links:
+
+ - [Website](https://www.amara.link/)
+ - [Docs](https://amara.gitbook.io/amara-finance/)
+ - [Github](https://github.com/AmaraFinance)
+ - [Medium](https://amara-finance.medium.com/)
+
+Contact us:
+
+ - [Discord](https://discord.com/invite/rhkyBmmCBf)
+
+ - [Twitter](https://twitter.com/AmaraFinance)
+
+ - [Telagram](https://twitter.com/AmaraFinance)
+
+ - [E-mail](mailto:amarafinance37@gmail.com)
+
+   
+
+## Moonbase Alpha Implementation
+
+AmaraLend Beta has been deployed on Moonbase Alpha, the testnet of Moonbeam, allowing depositing and lending of BTC, USDT, DOT, KSM. We are looking forward to having you with us. Access [here](https://lendtest.amara.link/)
+
+You can find a detailed tutorial on how to use AmaraLend beta: 
+
+https://amara.gitbook.io/amara-finance/
```

Comment:
Suggest:

```
[https://amara.gitbook.io/amara-finance/](https://amara.gitbook.io/amara-finance/)

Reply:
thu suggestions，we will improve the documentation